delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeView.scala

```diff
@@ -55,8 +57,8 @@ import scala.concurrent.duration.FiniteDuration
 final case class CompositeView(
     id: Iri,
     project: ProjectRef,
-    sources: NonEmptySet[CompositeViewSource],
-    projections: NonEmptySet[CompositeViewProjection],
+    sources: NonEmptyMap[Iri, CompositeViewSource],
```

Comment:
`NonEmptySet` were kind of a pain here and as we mostly want to access sources and projections individually, a `NonEmptyMap` makes more sense